config/scheduler/settings.ini

```diff
@@ -122,6 +122,7 @@ pa.scheduler.core.automaticremovejobdelay=0
 # This housekeeping feature can be replaced by a stored procedure
 # that runs at the desired period of time (e.g. non-business hours)
 # Such an example is available in samples/scripts/database/postgres/
+# Changing this setting is strongly not recommended as the support for pa.scheduler.job.removeFromDataBase=false has been discontinued
```

Comment:
So ... shouldn't be worthy to remove the support of this key completely ?

Reply:
Ideally, yes, but it spans so much of the API and tests that it is currently not feasible